src/types/api.d.ts

```diff
@@ -0,0 +1,28 @@
+interface MonitorDataLog {
+  date: string;
+  level: 'INFO' | 'WARNING' | 'ERROR';
+  description: string;
+}
+
+interface EnergySeries {
+  [key: string]: number;
+}
+
+interface MonitorData {
+  name: string;
+  has_error: boolean;
+  last_update: null | string;
+  energy_series: EnergySeries[];
+  logs: MonitorDataLog[];
+}
+
+interface Plant {
+  [key: string]: MonitorData;
+}
+
+export interface PollResponse {
+  done: boolean;
+  proc_start: string;
```

Comment:
Does Typescript have auto-conversion for ISO timestamps to Date or Timestamp types?
No big deal either way, just wondering if it might be useful.

Reply:
Nope, Typescript doesn't have such feature, as far as I know. 
I could create a type called ISO or something, but it would just be a string type with a different name. 
Should not be a problem either way as you said. 

More info here if you are curious: 
https://stackoverflow.com/questions/38359563/how-to-handle-iso-date-strings-in-typescript